prow/tide/gerrit.go

```diff
@@ -55,9 +55,24 @@ const (
 	// ref:
 	// https://gerrit-review.googlesource.com/Documentation/user-search.html#_search_operators.
 	// Also good to know: `(repo:repo-A OR repo:repo-B)`
-	gerritDefaultQueryParam = "status:open+-is:wip+is:submittable+label:" + tideEnablementLabel
+	gerritDefaultQueryParam = "status:open+-is:wip+is:submittable"
 )
 
+func gerritQueryParam(optInByDefault bool) string {
+	// Whenever a the `Prow-Auto-Submit` label is voted with -1 by anyone, the
+	// PR has to be excluded from Tide.
+	enablementLabelQueryParam := "+-label:" + tideEnablementLabel + "=-1"
+	// By default require `Prow-Auto-Submit` label.
+	// If the repo enabled optInByDefault, `Prow-Auto-Submit` is no longer
+	// required. But users can still temporarily opting out of merge automation
+	// by voting -1 on this label.
+	if !optInByDefault {
```

Comment:
NIt: Use `if optInByDefault` here instead of `!optInByDefault` because it's starting to get a little confusing which way the boolean goes.

Reply:
I'm not sure that I understand. The logic is that append query param when not optInByDefault, not sure how to easily do this the other way without duplicated logic

Reply:
I just meant [go/tott/530](https://goto.google.com/tott/530). If it complicates the code here (leading to more code duplication), then yeah don't bother.